css/main.css

```diff
@@ -65,3 +65,187 @@ header a {
   color: #172b4d;
   margin-bottom: 58px;
 }
+
+.container-2 {
+  margin-left: 26px;
+  margin-right: 22px;
+}
+
+.cricle {
+  width: 80px;
+  height: 80px;
+  margin-top: 40px;
+  border-radius: 100%;
+  background: white;
+  box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  row-gap: 24px;
+}
+
+.work-section h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.card {
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  width: 327px;
+}
+
+.project-details {
+  margin-top: 252px;
+  height: 222px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.project-details h3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0;
+}
+
+.project-details ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-evenly;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.project-details ul li {
+  background: #eff0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+.button-style {
+  padding: 12px;
+  background: #36b37f;
+  border-radius: 4px;
+  border: none;
+  color: white;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  letter-spacing: 0.03em;
+}
+
+.project-details button {
+  width: 122px;
+}
+
+.aboutme {
```

Comment:
- [ ] You have made a great effort in matching the Figma design 👍 , let's make it perfect by adding the background image in the about me section from the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design 

## Your Design 
![image](https://user-images.githubusercontent.com/60618877/176965508-a7649120-9291-4a23-8ce1-6529fc909a0d.png)


## Figma Design

![image](https://user-images.githubusercontent.com/60618877/176965353-34ff6406-9838-4c19-bef8-0c5bc9984f9f.png)

Reply:
@sumairq thanks a lot man I really forget about adding it thanks for reminding me I already did it thanks again.